src/containers/Employees/index.js

```diff
@@ -39,8 +39,10 @@ export default class Employees extends React.Component {
   };
 
   removeUser = slackID => {
-    console.log(slackID);
-    this.setState({ selectedMojos: [] });
+    const { selectedMojos } = this.state;
+    const mojo = selectedMojos.findIndex(user => user.slackID === slackID);
+    selectedMojos.splice(mojo, 1);
```

Comment:
This mutates the existing object which can result in unexpected behavior for objects in state. Check out ramda reject.

Reply:
`filter` and `reject` get to the same place from opposite directions. Up to you @jennyckaplan :)